_data/changelogs/component-modal.yml

```diff
@@ -2,6 +2,12 @@ title: Modal
 type: component
 changelogURL:
 items:
+- date: NNNN-NN-NN
+  summary: Removed deprecated xlink references.
+  summaryAdditional: All deprecated xlink:href references have been replaced with href.
+  affectsMarkup: true
+  githubPr: 6165
+  githubRepo: uswds
```

Comment:
**issue:** Missing USWDS version

```suggestion
- date: NNNN-NN-NN
  summary: Removed deprecated xlink references.
  summaryAdditional: All deprecated xlink:href references have been replaced with href.
  affectsMarkup: true
  githubPr: 6165
  githubRepo: uswds
  versionUswds: N.N.N
```

Reply:
Resolved in [93d5546](https://github.com/uswds/uswds-site/pull/2951/commits/93d554676a65798a6458efa3e93e7b5441596278)